Libraries/Components/TextInput/TextInput.js

```diff
@@ -532,6 +532,13 @@ export type Props = $ReadOnly<{|
    */
   editable?: ?boolean,
 
+  /** `readOnly` works like the `readonly` attribute in HTML.
+   *  If `true`, text is not editable. The default value is `false`.
+   *  See https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly
+   *  for more details.
+   */
+  readOnly?: ?boolean,
```

Comment:
It looks like we try to roughly follow alpha-sort for props, but if it's not being flagged by a linter it's not a problem.

Reply:
Good point, I've just updated it to be in the right order